plugin/core/transports.py

```diff
@@ -220,27 +245,37 @@ def _stderr_loop(self) -> None:
 
 
 # Can be a singleton since it doesn't hold any state.
-json_rpc_processor = JsonRpcProcessor()
+standard_processor = StandardProcessor()
+node_ipc_processor = NodeIpcProcessor()
```

Comment:
Again, I'm unsure whether this can actually be a singleton.

Reply:
Oh, right. The standard processor wasn't meant to have any instance properties so NodeIpc shouldn't either.